sky/authentication.py

```diff
@@ -65,18 +65,14 @@ def save_key_pair(private_key_path, public_key_path, private_key, public_key):
         f.write(public_key)
 
 
-def get_public_key_path(private_key_path):
-    if private_key_path.endswith('.pem'):
-        private_key_path, _ = private_key_path.rsplit('.', 1)
-    return private_key_path + '.pub'
-
-
-def get_or_generate_keys(private_key_path: str, public_key_path: str):
+def get_or_generate_keys():
     """Returns private and public keys from the given paths.
```

Comment:
Update doc string to reflect paths are no longer args?

Reply:
Good catch! Fixed. Thanks!